### PR TITLE
Update Javadocs/slice2html URLs to point at downloads.openmicroscopy.org

### PR DESCRIPTION
**Pull request by sbesson**
The slice2html CSS as well as image resources for the Javadocs/slice2html
documentation were previously hosted on ci.openmicroscopy.org. This commit
migrates this URL to downloads.openmicroscopy.org where this documentation
is currently hosted.

Additionally this PR gets rid of the custom Javadoc CSS stylesheet meaning the OMERO API Javadoc will inherit the stylesheet of the Java version used for compilation.

### CODE REVIEW

File: docs/styleForAPI.css [hunk 1]
```diff
@@ -1,189 +0,0 @@
...
-
-#SymbolTree {
-	font-size: larger;
-	font-variant: normal;
-	list-style-position: outside;
-}
-
-#banner {
-	background-color: #000066;
-	background-image: url(http://hudson.openmicroscopy.org.uk/userContent/ome_trac_back_top.png);
```
Comment (mtbc) [inline docs/styleForAPI.css]: http://downloads.openmicroscopy.org/resources/styleForAPI.css has this too, will that also need updating?

Comment (sbesson) [inline docs/styleForAPI.css]: Good point. Done.

Comment (mtbc) [inline docs/styleForAPI.css]: :+1:

**Comment by sbesson** [comment]
Last commit also drops the custom Javadocs CSS to match the default style using by Bio-Formats and remove maintenance.

**Comment by sbesson** [comment]
See https://ci.openmicroscopy.org/view/DEV/job/OMERO-DEV-merge-build/ICE=3.5,jdk=8_LATEST,label=octopus/javadoc/. One more commit to come to address the CSS in the `slice2html` docs.

**Comment by sbesson** [comment]
https://ci.openmicroscopy.org/view/DEV/job/OMERO-DEV-merge-build/ICE=3.5,jdk=8_LATEST,label=octopus/javadoc/ should now inherit the default Javadoc style of the Java used for compiling (Java 8 in this case) and https://ci.openmicroscopy.org/view/DEV/job/OMERO-DEV-merge-build/ICE=3.5,jdk=8_LATEST,label=octopus/javadoc/slice2html/index.html uses the resources from downloads.

**Comment by joshmoore** [comment]
Barring the CSS changes, everything makes sense. You expect changes in the javadocs on CI, correct? What about when released to the downloads page?

**Comment by sbesson** [comment]
So summarizing the current CSS status for the OMERO Javadocs
- https://ci.openmicroscopy.org/view/DEV/job/OMERO-DEV-latest/ICE=3.5,jdk=8_LATEST,label=octopus/ uses the custom OMERO Javadoc CSS file which removal is proposed to in this PR
-  https://ci.openmicroscopy.org/view/DEV/job/OMERO-DEV-merge-build/ICE=3.5,jdk=8_LATEST,label=octopus/ uses the same stylesheet as the Java version used for building (https://docs.oracle.com/javase/8/docs/api/index.html in this case)
- the downloads page would also use the sylesheet of the Java version used when building (as it is already the case for http://downloads.openmicroscopy.org/bio-formats/5.1.4/api/ which inherits the stylesheet from https://docs.oracle.com/javase/7/docs/api/index.html)

**Comment by joshmoore** [comment]
I.e. we lose the banner. It's nice, but not worth any trouble.

**Comment by hflynn** [comment]
Seconding the banner not being worth any trouble, the other style sheet is perfectly readable. This is good to go from my POV.

**Pull request merged**